control-plane/roles/isolated-clusters/templates/ingress/values.yaml

```diff
@@ -0,0 +1,26 @@
+controller:
+  image:
+    chroot: {{ isolated_clusters_ingress_ingress_controller_chroot | default(false) }}
+
+  replicaCount: {{ isolated_clusters_ingress_ingress_controller_replicas | default(1) }}
+  extraArgs:
+    enable-ssl-passthrough: ""
```

Comment:
Is there a specific reason we need to enable SSH passthrough?

Reply:
Can probably be deleted